ddl/index.go

```diff
@@ -467,13 +471,16 @@ type addIndexWorker struct {
 	index       table.Index
 	table       table.Table
 	colFieldMap map[int64]*types.FieldType
+	colExprMap  map[int64]expression.Expression
 	closed      bool
 	priority    int
 
 	// The following attributes are used to reduce memory allocation.
 	defaultVals        []types.Datum
 	idxRecords         []*indexRecord
 	rowMap             map[int64]types.Datum
+	rowSlice           []types.Datum
```

Comment:
`colSlice` instead of `row`?

Reply:
`colSlice` is just a buffer, main purpose is  getting `row` for expression `eval`.

Reply:
I think `rowBuffer` is a more clear name.